### DISCUSSION TITLE
Fomat: Semicolons > remove adds null

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [BowlerHatLLC/vscode-as3mxml#737] [Q&A] [answered]
![image](https://github.com/BowlerHatLLC/vscode-as3mxml/assets/10191085/c7d93ed2-3bf6-47ae-a947-1a8b024f072e)

Formatter not removing semicolons, but adds null instead:

![image](https://github.com/BowlerHatLLC/vscode-as3mxml/assets/10191085/566b0537-0daa-425a-a8a9-3bc84dacabd9)

**Comment by joshtynjala** [comment]
Interesting! I can reproduce this bug. I'll investigate.

**Reply by joshtynjala** [reply, marked as answer]
It looks like this bug was already fixed in apache/royale-compiler. The fix will be included in vscode-as3mxml after the next Royale release.

apache/royale-compiler@1c250eb30ececdf48fdf07783c065c9770117301
